jdaviz/core/helpers.py

```diff
@@ -378,11 +382,325 @@ def show_in_sidecar(self, anchor=None, title=None):
         location = 'sidecar' if anchor is None else f"sidecar:{anchor}"
         return self.show(loc=location, title=title)
 
-    def show_in_new_tab(self, title=None):
+    def show_in_new_tab(self, title=None):  # pragma: no cover
         """
         Preserved for backwards compatibility
         Shows Jdaviz in a sidecar in a new tab to the right
         """
         warnings.warn('show_in_new_tab has been replaced with show(loc="sidecar:tab-after")',
                       DeprecationWarning)
         return self.show(loc="sidecar:tab-after", title=title)
+
+
+class ImageConfigHelper(ConfigHelper):
+    """`ConfigHelper` that uses an image viewer as its primary viewer.
+    For example, Imviz and Cubeviz.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # NOTE: The first viewer must always be there and is an image viewer.
+        self._default_viewer = self.app.get_viewer_by_id(f'{self.app.config}-0')
+
+    @property
+    def default_viewer(self):
+        """Default viewer instance. This is typically the first viewer
+        (e.g., "imviz-0" or "cubeviz-0")."""
+        return self._default_viewer
```

Comment:
what are the implications/restrictions for an image viewer needing to be the primary/default_viewer?  Is this something we'll need to think about if also bringing this to mosviz down the road?

Reply:
`default_viewer` existed for Imviz and now Cubeviz needs this so I can port the features over without breaking Imviz API.